index.html

```diff
@@ -1,27 +1,166 @@
+                        creating a positive change with people from over 80 countries
+                        is taking place in September in Swaziland
+                    </p>
+                </div>
+                <h3 class="summit-date">2023.08.15(THUR) ~ 16(FRI)</h3>
+                <p class="summit-venue">@ National Museum of eSwatini, Art Center Ezulwini and more</p>
+            </div>
+        </div>
+    </section>
+
+    <!-- =========Main Program Section========= -->
+    <section class="main-program mb-2">
+        <div class="main-program-title">
+            <p>Main Program</p>
+            <hr class="hr-line">
+        </div>
+        <div class="programs-card-container mb-3 pt-3">
+            <div class="program">
+                <img src="img/programIcons/program-icon-1.png" alt="#" class="program-img">
+                <p class="program-title">Lecture</p>
+                <p class="program-description">Listen to speaker from various countries about the messages of sharing
+                    and opening.</p>
+            </div>
+            <div class="program">
+                <img src="img/programIcons/program-icon-2.png" alt="#" class="program-img">
+                <p class="program-title">CS Exhibition</p>
+                <p class="program-description">Appreciate various creations Applying CC license of artists, organized
+                    from art center Nabi.</p>
+            </div>
+            <div class="program">
+                <img src="img/programIcons/program-icon-3.png" alt="#" class="program-img">
+                <p class="program-title">Forum</p>
+                <p class="program-description">Have time to share your thoughts and opinions with experts for each
+                    topic.</p>
+            </div>
+            <div class="program">
+                <img src="img/programIcons/program-icon-4.png" alt="#" class="program-img">
+                <p class="program-title">Workshop</p>
+                <p class="program-description">Try creating your own work using open source license rather than just
+                    watching at it.</p>
+            </div>
+            <div class="program">
+                <img src="img/programIcons/program-icon-5.png" alt="#" class="program-img">
+                <p class="program-title">CS Ignite</p>
+                <p class="program-description">Get opportunities to network with CC Affiliates around the worls, also
+                    after the summit.</p>
+            </div>
+        </div>
+        <div class="see-the-whole-program mt-5 mb-5">
+            <p>
+                <a href="#">See the whole program</a>
+            </p>
+        </div>
+
+        <button class="join-cs mb-3">Join CS African Summit 2021</button>
+    </section>
+
+    <!-- ===========Featured Section============= -->
+    <section id="featured-speakers" class="featured-speakers mb-4 py-4">
+
+    </section>
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: your feature speaker section is still overflowing kindly fix it. Please try to keep every content in its section. This will make your webpage easy to navigate. :slightly_smiling_face: 

![Screenshot from 2023-04-29 17-05-10](https://user-images.githubusercontent.com/45226848/235302411-cef9903c-baf0-46d6-9a90-4ed25c9f034b.png)